bridgestyle/arcgis/togeostyler.py

```diff
@@ -140,10 +140,15 @@ def processLabelClass(labelClass, tolowercase=False):
     maplexProperties = labelClass.get("maplexLabelPlacementProperties", {})
     maplexPlacementType = maplexProperties.get("featureType")
     maplexPrimaryOffset = maplexProperties.get("primaryOffset", 0)
+    maplexPointPlacementMethod = maplexProperties.get("pointPlacementMethod")
```

Comment:
doesn't this require a default value? how does the `==` below work if it is a NoneType object?

Reply:
Thanks for checking. If the key does not exist the default return value of `get()` is `None` so the `==` is evaluated to false. I think this is OK